spec/graphql/schema/argument_spec.rb

```diff
@@ -82,6 +82,13 @@ def context_arg_test(input:)
       end
 
       field :other_unauthorized_instruments, resolver: LoadUnauthorizedInstruments
+
+      # Warden's answer for `possible_types(UnauthorizedInstrumentType)` was an empty array
+      # because the type wasn't used in the schema except as `loads:...`
+      # This makes Subset and Warden behave the same.
+      field :example_unauthorized_instrument, UnauthorizedInstrumentType
+      # Same thing:
+      field :example_instrument, Jazz::InstrumentType
```

Comment:
Here's another kind of divergence I had to address. 

- When values are loaded with `loads:`, GraphQL-Ruby confirms that they match the expected type by doing `possible_types(loadable_type).include?(resolve_type(loaded_object))`. So, `possible_types(...)` may be called on plain object types.
- `Warden`  hid types from `possible_types` if they weren't attached as field return types. So in cases like this, it returned `[]`
- But `Subset` avoids that traversal if possible, so it returns `[obj_type]` without traversing. 

This divergence raised an error with TypesMigration, so I added this field to address it. 

If this becomes a problem, and people can't attach fields to the schema like this, we might be able to somehow track `@loads_only_types` in Subset to handle cases like this.

Reply:
Improved `loadable?` tracking in ae4874452